src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -98,6 +110,13 @@ export class NetworkAreaDiagramViewer {
     originalTextNodeConnectionShift: Point = new Point(0, 0);
     lastZoomLevel: number = 0;
     zoomLevels: number[] = [0, 1000, 2200, 2500, 3000, 4000, 9000, 12000, 20000];
+    bendLines: boolean = false;
+    onBendLineCallback: OnBendLineCallbackType | null;
+    straightenedElement: SVGGraphicsElement | null = null;
+    bendableLines: string[] = [];
+
+    linePointIndexMap = new Map<SVGGElement, { edgeId: string; index: number }>();
+    linePointByEdgeIndexMap = new Map<string, SVGGElement>();
```

Comment:
I think we may delete this map at a slight performance cost. The update bent line code would be much simpler and would rely simply on the first map:
```java
this.linePointIndexMap.forEach((value) => {
    if (value.edgeId == edge.svgId && value.index >= linePoints.index) {
        value.index++; // or value.index-- in the straighten case
    }
});
```

To avoid this cost looping over all the values of the map, we would need to have two simple maps, one map giving simply the edge id based on the line point id, and the other one being a Map<string, Map<string, index>> giving the map of indexes for each edge id.